golem-worker-service-base/src/gateway_api_definition/http/api_definition.rs

```diff
@@ -82,29 +82,31 @@ impl HttpApiDefinition {
     ) -> Result<Self, ApiDefinitionError> {
         let mut registry = HashMap::new();
 
-        if let Some(security_schemes) = request.security {
-            for security_scheme_reference in security_schemes {
-                let security_scheme = security_scheme_service
-                    .get(
-                        &security_scheme_reference.security_scheme_identifier,
-                        namespace,
-                    )
-                    .await
-                    .map_err(ApiDefinitionError::SecuritySchemeError)?;
+        let security_schemes_in_definition = request
+            .routes
```

Comment:
The outer level security was not helping users before, and acted as something _to keep it as close to open-api-spec_. But I have decided to remove it, and we can still generate lossless open-api spec. Only route level security is needed (and that's all we documented too)